series-tvdb.en.yaml

```diff
@@ -501,6 +501,15 @@ entries:
       - season: 2
         anilist-id: 18115
 
+  - title: "Masou Gakuen HxH"
+    synonyms:
+      - "Magias Academy Ataraxia Hybrid x Heart"
+    seasons:
+      - season: 0
```

Comment:
Season 0 is not supported

Reply:
Fixed in 35d961f35458efcbd792e45fd16c49898262c906